lib/wings/hydra/works/services/add_file_to_file_set.rb

```diff
@@ -37,9 +37,9 @@ def association_type(type)
         end
 
         def type_to_association_type(type)
-          return :original_file if type.to_s.casecmp?(Hyrax::FileSet::ORIGINAL_FILE_USE.to_s)
-          return :extracted_text if type.to_s.casecmp?(Hyrax::FileSet::EXTRACTED_TEXT_USE.to_s)
-          return :thumbnail if type.to_s.casecmp?(Hyrax::FileSet::THUMBNAIL_USE.to_s)
+          return :original_file if type.to_s.casecmp?(Hyrax::FileMetadata::Use::ORIGINAL_FILE.to_s)
```

Comment:
these probably don't need `casecmp?` either? is `type` going to be an `RDF::URI`?

Reply:
yeah, agreed on all points. i'm thinking the issue of the symbols is out-of-scope for now.